packages/assets-controllers/src/AssetsContractController.test.ts

```diff
@@ -45,6 +46,7 @@ const setupControllers = () => {
   });
   const preferences = new PreferencesController();
   const assetsContract = new AssetsContractController({
+    chainId: ChainId.mainnet,
```

Comment:
Ah I see. I wasn't considering this as a default because it's just a unit test.

Expressing that this is a default is a good idea though. There are some planned refactors that would fit nicely with that idea. This whole setup function is something we'd like to replace, so that the initial constructor parameters can be set per-test rather than hard coded here.